test/integration/modules/edgecore/eventbus/eventbus_suite_test.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2019 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"crypto/tls"
+	"io"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/kubeedge/kubeedge/test/integration/utils/common"
+	"github.com/kubeedge/kubeedge/test/integration/utils/edge"
+
+	MQTT "github.com/eclipse/paho.mqtt.golang"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+//context to load config and access across the package
+var (
+	ctx *edge.TestContext
+	cfg edge.Config
+)
+
+//Interface to validate the MQTT connection.
+type Token interface {
+	Wait() bool
+	WaitTimeout(time.Duration) bool
+	Error() error
+}
+
+const (
+	//deviceupload topic
+	DeviceUpload = "$hw/events/upload/#"
```

Comment:
we can reuse the variables defined in pkg\devicetwin\dtcommon\common.go instead of defining them as constants here again.

Reply:
@rohitsardesai83, pkg\devicetwin\dtcommon\common.go doesnt have these subscribe topics defined. as per your suggestion i supposed add these const to common.go in devicetwin module.

i feel modfying the source code for integration test is not required, thats why i defined in integration package intself.

Reply:
They are declared in eventbus/mqtt/client.go in an array. Can it be reused?

Reply:
No, im reusing the dtcommon variables itself here.